stores/metadata.go

```diff
@@ -2685,20 +2685,32 @@ func archiveContracts(ctx context.Context, tx *gorm.DB, contracts []dbContract,
 	return nil
 }
 
+func pruneSlabs(tx *gorm.DB) error {
+	// delete slabs without any associated slices or buffers
+	return tx.Exec(`
+DELETE
+FROM slabs
+WHERE NOT EXISTS (SELECT 1 FROM slices WHERE slices.db_slab_id = slabs.id)
+AND slabs.db_buffered_slab_id IS NULL
```

Comment:
just checking to make sure here, is this covering the `AND slices.db_multipart_part_id IS NULL` case?

Reply:
```sql 
WHERE NOT EXISTS (SELECT 1 FROM slices WHERE slices.db_slab_id = slabs.id)
```
should only be `true` if no slices exist that reference a given slab. Which means there can also not be any slices where `slices.db_multipart_part_id IS NULL` is `true` for.